app/src/components/UI/Atoms/Map.tsx

```diff
@@ -66,9 +68,10 @@ export default function Map(): React.ReactElement {
       center={[47.584401, -122.14819]}
       zoom={14}
       style={{
-        height: "92.3vh",
-        width: "100%",
+        height: "90.3vh",
+        width: "180%",
```

Comment:
The same problem still exists. You should not be setting a percentage relative to the parent element, as the parent element is independent of the window size. The width value should be relative to the window width.
![Screenshot from 2022-02-17 10-10-00](https://user-images.githubusercontent.com/36318069/154544431-ebfda5c1-abb2-46a3-a504-f4b682198751.png)

Reply:
I fixed it like this. Using React devtools, I checked which components were sized smaller than the browser width and set them to max width.

```diff
diff --git a/app/src/components/UI/Atoms/Map.tsx b/app/src/components/UI/Atoms/Map.tsx
index 2bf3d00..af2ecb9 100644
--- a/app/src/components/UI/Atoms/Map.tsx
+++ b/app/src/components/UI/Atoms/Map.tsx
@@ -69,9 +69,8 @@ export default function Map(): React.ReactElement {
       zoom={14}
       style={{
         height: "90.3vh",
-        width: "180%",
+        width: "100%",
         marginTop: 2.9,
-        marginLeft: -425,
         zIndex: 0,
       }}
     >
diff --git a/app/src/components/UI/Pages/LandingPage.tsx b/app/src/components/UI/Pages/LandingPage.tsx
index 4adcc4a..f545c3f 100644
--- a/app/src/components/UI/Pages/LandingPage.tsx
+++ b/app/src/components/UI/Pages/LandingPage.tsx
@@ -20,7 +20,7 @@ export const LandingPage = () => {
       <Sidebar.Pushable as={Container} className={styles.container}>
         <LandingPageSidebar setVisible={setVisible} visible={visible} />
         <Sidebar.Pusher>
-          <Container>
+          <Container fluid>
             <Button
               icon
               className={styles.icon}
diff --git a/app/src/components/UI/Pages/landingpage.module.css b/app/src/components/UI/Pages/landingpage.module.css
index a354ba0..1c75acb 100644
--- a/app/src/components/UI/Pages/landingpage.module.css
+++ b/app/src/components/UI/Pages/landingpage.module.css
@@ -9,7 +9,3 @@
   z-index: 100 !important;
   cursor: pointer;
 }
-
-.container {
-  width: 150% !important;
-}

```

Reply:
I believe there is a container that is not set to `fluid`.

Reply:
![image](https://user-images.githubusercontent.com/42430035/154555001-8eafc278-e800-4d9f-a4b1-fcd540283559.png)
Oh wait, I changed this line and it seems to work now!